### PR TITLE
Update dependencies & fix timeout_test.dart

### PR DESCRIPTION
**Pull request by SaadArdati**

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -9,10 +9,10 @@ environment:
   sdk: '>=2.12.0 <3.0.0'
 
 dependencies:
-  buffer: ^1.0.0
+  buffer: ^1.2.2
```
Comment (isoos) [inline pubspec.yaml:12]: note: I don't think this is strictly necessary, but would not hurt in this case.

**Comment by isoos** [comment]
Could you also please bump the version in `pubspec.yaml` and write a small note in `CHANGELOG.md`?

**Comment by SaadArdati** [comment]
@isoos Done.

**Review by isoos — APPROVED** [review]

**Comment by isoos** [comment]
thanks, published.

**Pull request merged**